inc/apirest.class.php

```diff
@@ -55,6 +55,23 @@ public static function getTypeName($nb = 0) {
       return __('Rest API');
    }
 
+   /**
+    * Upload and validate files from request and append to $this->parameters['input']
+    *
+    * @return void
+    */
+   public function manageUploadedFiles() {
```

Comment:
last thing (sorry)
make this function abstract in api.class, and do it also in xmlrpc (empty of course)